src/index.css

```diff
@@ -60,32 +60,50 @@ code {
   }
 }
 
+/* Dropzone styles */
+
 .dropzone {
+  height: 80px;
   flex: 1;
   display: flex;
   flex-direction: column;
   align-items: center;
-  min-width: 60%;
-  padding: 20px;
-  border-width: 2px;
+  justify-content: center;
+  border-width: 1.5px;
   border-radius: 2px;
-  border-color: #eeeeee;
+  border-color: var(--color-gray-light);
   border-style: dashed;
-  background-color: #ffffff;
-  color: #bdbdbd;
   outline: none;
   cursor: pointer;
   transition: border .24s ease-in-out;
 }
 
 .dropzone-active {
-  border-color: #2196f3;
+  border-color: var(--color-primary);
 }
 
 .dropzone-accepted {
-  border-color: #2ED8A3;
+  border-color: var(--color-primary);
 }
 
 .dropzone:focus {
-  border-color: #2196f3;
+  border-color: var(--color-primary);
+}
+
+/* Modals styles */
+
+.modalContainer {
+  display: flex;
+  width: 700px;
+}
+
+.modalIconRow {
+  padding-right: 20px;
+}
+
+.modalInfoText {
+  color: var(--color-gray-dark);
+  padding-top: 20px;
+  padding-right: 80px;
+  text-align: left;
```

Comment:
This shouldn't be here. You can create custom components for this.

Reply:
Fixed